functions/public/Invoke-WPFUpdateMGMGT.ps1

```diff
@@ -0,0 +1,63 @@
+function Invoke-WPFUpdateMGMT {
+    param (
+        [switch]$Selected,
+        [switch]$All
+    )
+
+    if ((-not $Selected -and -not $All) -or ($Selected -and $All)) {
+        Write-Host "[Invoke-WPFUpdateMGMT] Use either 'Selected' or 'All' switches, used switches are: 'Selected' is $Selected, 'All' is $All" -ForegroundColor Red
+        return
+    }
+
+    $sync.form.Dispatcher.Invoke([action]{ Set-WinUtilTaskbaritem -state "Indeterminate" -value 0.01 -overlay "logo" })
+
+    if ($All) {
+        Write-Host "Installing all available updates ..."
+        Invoke-WPFRunspace -ArgumentList $sync["WPFUpdateVerbose"].IsChecked -DebugPreference $DebugPreference -ScriptBlock {
+            param ($WPFUpdateVerbose)
+            if ($WPFUpdateVerbose) {
+                Install-WindowsUpdate -Verbose -Confirm:$false -IgnoreReboot:$true -IgnoreRebootRequired:$true
+            } else {
+                Install-WindowsUpdate -Confirm:$false -IgnoreReboot:$true -IgnoreRebootRequired:$true
+            }
+            $sync.form.Dispatcher.Invoke([action]{ Set-WinUtilTaskbaritem -state "None" -overlay "checkmark" })
+            Write-Host "All Update Processes Completed"
```

Comment:
Currently neither the install all, nor install selected code handles updating the DataGrid (Updates List) according to the result of installation (if successful, remove the update(s) from the list, otherwise keep them) which'll help visualize the remaining updates. Of course an easier solution is to call `Invoke-WPFUpdateScan` that gets all of the updates (for the third time, first two is done by user when clicking the button, and the second by install code), and also update DataGrid.